x/qvesting/keeper/grpc_query.go

```diff
@@ -65,44 +143,29 @@ func (k Keeper) SpendableBalances(ctx context.Context, req *types.QuerySpendable
 	}, nil
 }
 
-func (k Keeper) VestingAccounts(c context.Context, req *types.QueryVestingAccountsRequest) (*types.QueryVestingAccountsResponse, error) {
+// SpendableSupply provides an aggregated overview of the total of the spendable balances across all accounts for a given denom.
+func (k Keeper) SpendableSupply(ctx context.Context, req *types.QuerySpendableSupplyRequest) (*types.QuerySpendableSupplyResponse, error) {
```

Comment:
Look like it is iterating over only vesting accounts and summing the spendable supply of all vesting accounts only?
How does this method take care of the non vesting accounts balances?
total spendable (float) should be equals to the sum of spendables for both vesting and non vesting accounts.